lib/rspec_profiling/collectors/statsd.rb

```diff
@@ -146,7 +151,8 @@ def insert(attributes)
         stamp = build_stamp(attributes.fetch(:description), attributes.fetch(:line_number))
         path = format_file(attributes.fetch(:file))
         branch = attributes.fetch(:branch)
-        key = "#{branch}.#{hash}.#{path}.#{stamp}".gsub("\n", '')
+        commit_date = format_date(attributes.fetch(:date))
```

Comment:
If you want to merge your fork changes into main repo, such conversions should be optional and configurable IMO

Reply:
There is no plan to merge back into the main repo, mainly due to the mass removal of the SQL collectors etc.